avocado/core/plugin_interfaces.py

```diff
@@ -424,6 +424,17 @@ async def update_requirement_cache(runtime_task, result):
         :type result: `avocado.core.teststatus.STATUSES`
         """
 
+    @staticmethod
+    @abc.abstractmethod
+    def is_operational():
```

Comment:
Yes, but I meant that you never call it as static method. This is used in avocado/plugins/runner_nrunner.py where you're initializing the spawner object. Therefore, IMO this method doesn't have to be a static method and it will fix the pylint W0221 warning on `PodmanSpawner`.

Reply:
If this method is obligatory for all spawners is there any chance we can at least add something like
```python
    @staticmethod
    def is_operational():
        return True
```
to the LXC spawner within the same pull request, i.e. on the same ground as both the process and podman spawners?

Reply:
I'll change `is_oeprational()` to be an instance method.  The fact that the podman spawner needs that, is a good enough "hint" that it's the better choice indeed.